docs/04-operation-guides/operations/10-backup-kyma.md

```diff
@@ -1,29 +1,232 @@
 
-Kyma relies on a managed Kubernetes cluster for periodic backups of Kubernetes objects to avoid any manual steps.
+### Object backup
 
->**CAUTION:** Automatic backup does not include Kubernetes volumes. Back up your volumes periodically either on demand, or set up a periodic job.
+Kyma relies on a managed Kubernetes cluster for periodic backups of Kubernetes objects to avoid any manual steps.
 
-## Object backup
+>**CAUTION:** Automatic backup doesn't include Kubernetes volumes. Back up your volumes periodically either on demand, or set up a periodic job.
 
 For example, Gardener uses [etcd](https://etcd.io/) as the Kubernetes backing store for all cluster data. Gardener runs periodic jobs to take major and minor snapshots of the etcd database to include Kubernetes objects in the backup.
 
 The major snapshot that includes all resources is taken on a daily basis, while minor snapshots are taken every five minutes.
 
 If the etcd database experiences any problems, Gardener automatically restores the Kubernetes cluster using the most recent snapshot.
 
-## Volume backup
+### Volume backup
+
+We recommend that you back up your volumes periodically with the [VolumeSnapshot API resource](https://kubernetes.io/docs/concepts/storage/volume-snapshots/#volumesnapshots), which is provided by Kubernetes. You can use your snapshot to provision a new volume prepopulated with the snapshot data, or restore the existing volume to the state represented by the snapshot.
+
+Taking volume snapshots is possible thanks to [Container Storage Interface (CSI) drivers](https://kubernetes-csi.github.io/docs/), which allow third-party storage providers to expose storage systems in Kubernetes. For details on available drivers, see the [full list of drivers](https://kubernetes-csi.github.io/docs/drivers.html).
+
+You can create on-demand volume snapshots manually, or set up a periodic job that takes automatic snapshots periodically.
+
+## Back up resources using Velero
+
+You can back up and restore individual resources manually or automatically with Velero. For more information, read the [Velero documentation](https://velero.io/docs/).
+Be aware that a full backup of a Kyma cluster isn't supported. Start with the existing Kyma installation and restore specific resources individually.
+
+## Create on-demand volume snapshots
+
+If you want to provision a new volume or restore the existing one, create on-demand [volume snapshots](https://kubernetes.io/docs/concepts/storage/volume-snapshots/):
+
+<div tabs name="backup-providers">
+  <details>
+  <summary label="AKS">
+  AKS
+  </summary>
+
+### Prerequisites
+
+  The minimum supported Kubernetes version is 1.17.
+
+### Steps
+
+  1. [Install the CSI driver](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/docs/install-csi-driver-master.md).
```

Comment:
It is a broken link.

Reply:
Weird, it works from github and when I copy + paste the URL. It's just broken in the netlify preview.  How???

Reply:
Investigation with https://sap-btp.slack.com/archives/C01RAHB7D25/p1627558780003700?thread_ts=1627468836.000800&cid=C01RAHB7D25